content/posts/building-images-with-img.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Building Container Images with img
```

Comment:
Suggest possibly adding Kubernetes to the title - as that is a big part of this post: "Building Container Images on Kubernetes with img"

Reply:
Updated to "Unprivileged Container Image Builds with img and Jenkins on Kubernetes" to include a few more descriptors - let me know your thoughts there.